src/components/ads/AdHandling.js

```diff
@@ -125,26 +121,7 @@ const AdHandler = () => {
                     </div>
                 </div>
             )}
-             {/* Pagination Controls */}
```

Comment:
removed pagination per request

Reply:
Perfect, we will have more user friendly UI